### ISSUE TITLE
deisctl gets to "failed/failed" on a service, and then hangs.

### ISSUE THREAD

**Issue by technosophos** [deis/deis#3880]
On occasion, if I get a service to exit unexpectedly, then deisctl will hang on any subsequent commands.

Example:

```
$ deisctl stop builder
deis-builder.service: failed/failed
(hangs)
```

With a little bit of debugging code in `fleet/stop.go`, I can see that it's stuck in the for loop there:

```go
		lastSubState = currentState.SystemdSubState
		fmt.Printf("Waiting for state %s, but in state %s\n", desiredState, currentState.SystemdSubState) // Added
		time.Sleep(250 * time.Millisecond)
```

Which prints this out every 250 msec:

```
Waiting for state dead, but in state failed
```

I'm not sure if we're legitimately waiting for something to happen, or if `failed` or `dead` should be treated like a special case.

**Comment by bacongobbler** [comment]
This has been an issue with deisctl since inception. It's probably a mis-use of the fleet client API that's causing this issue. IIRC the idea is that you give the fleet API a DesiredState (in this case, `loaded/dead`) and deisctl is just looping and waiting for that unit to show that it's now in that desired state.

**Comment by mboersma** [comment]
Closed by #4095.
